### DISCUSSION TITLE
Faulty help link on General configuration page

### DISCUSSION THREAD

**Discussion by HermanGyorgy** [Yvand/EntraCP#217] [General]
Installed v23.0.20231121.30 to get familiar with.
Found the following NON functionality issue:
On the "Global configuration" (_admin/EntraCP/GlobalSettings.aspx) page below the "Register a new Microsoft Entra ID tenant" section there is a "this page" link that points to:  
https://entracp.yvand.net/docs/usage/register-application/
Trying to access this link browser gives HTTP 404.
I think the link should point to the below url instead, where I see the related content:
https://entracp.yvand.net/overview/register-application/

**Comment by Yvand** [comment]
Many thanks for reporting this!
